db/seeds.rb

```diff
@@ -5,3 +5,4 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
```

Comment:
- [ ] Kindly add 5 messages to your app, this is stated in the project requirements, and the only way you can do this is by creating those messages using the seed file.